tests/baselines/reference/superInConstructorParam1.js

```diff
@@ -27,7 +27,7 @@ var B = (function () {
 var C = (function (_super) {
     __extends(C, _super);
     function C(a) {
-        if (a === void 0) { a = _super.prototype.foo.call(this); }
+        if (a === void 0) { a = _super.foo.call(this); }

```

Comment:
I assume we detect this as an error? It seems like `_super.prototype.foo` would be the correct emit, though it may not matter if it is an error.